CHANGELOG.md

```diff
@@ -51,14 +54,18 @@
 - Improved error experience of `Session.write_pandas` method that `NotImplementError` will be raised when called.
 - Aligned error experience with reusing a closed session in non-local execution.
 
-## 1.16.0 (TBD)
+## 1.16.0 (2024-05-07)
 
 ### New Features
 
 - Support stored procedure register with packages given as Python modules.
 - Added snowflake.snowpark.Session.lineage.trace to explore data lineage of snowfake objects.
 - Added support for structured type schema parsing.
 
+### Bug Fixes
+
+- Fixed a bug when inferring schema, single quotes are added to stage files already have single quotes.
```

Comment:
Fixed a bug where, when inferring a schema, single quotes were added to stage files that already had single quotes.